code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -799,7 +799,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 	tnote += signal
 
 	if (!silent)
-		if(HAS_TRAIT(SSstation, STATION_TRAIT_PDA_GLITCHED))
+		if(HAS_TRAIT(SSstation, STATION_TRAIT_PDA_GLITCHED) && can_play_alt_sound())
```

Comment:
Why is the alt PDA sound based solely on the prefs of the person holding the PDA?

Reply:
hm, good idea, I could make an override in playsound

Reply:
i made it override using a list in playsound instead